file_formats/nwb.md

```diff
@@ -0,0 +1,62 @@
+# Standards on Neurodata Without Borders files
+
+## Version
+
+`0.1.0-draft`
+
+## Introduction
+
+Neurodata Without Borders (NWB) files are a standard file format for neurophysiology data, however there are many topics where NWB is not opinionated. This documents defines AIND's opinions. 
+
+## Basics
+
+We also take this opportunity to remind readers about some general NWB basics:
+
+- Raw data goes in the `acquisition` group
+- Processed data goes in the `processing` group
+- The root level `session_start_time` is a timezone-aware datetime defining the start of acquisition.
+- All timestamps must relative to the same point in time and require no additional alignment to compare. This is defined in the root level `timestamps_reference_time`, which defaults to `session_start_time`.
+- It is not required that timestamps for different data streams be identical (i.e. resample data such that all timeseries to have exactly the same timestamps).
+
+## HARP
+
+- At AIND, if there is a HARP board available, timestamps must utilize HARP TTLs to transform timestamps to a common timebase. It is insufficient to store a lookup table.
+
+## Events 
+
+- Any events should be packaged using the [ndx-events](https://github.com/rly/ndx-events) NWB extension.
+- We represent all events in an `EventTable` and all event descriptions in a `MeaningsTable` table.
+- Events have a `timestamp` property, and arbitrary property names and corresponding values.
+- Property values can have `meaning`s (e.g. property name "lick" with value "0" means "the mouse licked the left water port").
+- All event properties and values must be described by [HED tags](https://www.hedtags.org/) or part of AIND's HED extension.
```

Comment:
We should get a general consensus from behavior/SWE folks of what this effort entails. A different way to go about this may be to start with a warning-level qc that simply flags events without a valid HED tag name and give people time to slowly adopt this standard. There is a considerable ONGOING cost to adopt these tags. If every new event that we add must have an HED tag it will always require changing things in multiple places.

Reply:
I'm fine with making HED optional for a period of time as long as progress is being made on a clear timescale.